force-app/main/default/lwc/accountModelSettings/accountModelSettings.js

```diff
@@ -14,8 +25,56 @@ export default class AccountModelSettings extends LightningElement {
 
     labelReference = {
         stgAccountModelSettingsTitle,
+        stgAccModelTitle,
+        stgHelpAccountModel,
+        stgAdminAccountRecordType,
+        stgHelpAdminRecType,
+        stgAccountRecordTypeSupportsHHAddress,
+        stgHelpHouseholdRecType,
     };
 
+    @wire(getAccountModelSettingsViewModel)
+    accountModelSettingsViewModel({ error, data }) {
+        if (data) {
+            this.accountModelSettingsViewModel = data;
+        } else if (error) {
+            console.log("error retrieving accountmodelsettingsvmodel");
+        }
+    }
+
+    handleDefaultAccountModelChange(event) {
+        // add updated setting to hierarchySettingsChanges object
+        let hierarchySettingsChange = {
+            settingsType: "string",
+            settingsName: "Account_Processor__c".toLowerCase(),
```

Comment:
Do we need lower case here? Legitimate question, I'm not sure either way, but if we don't need to process this further, then let's not.